tests/syntax-tests/highlighted/fish_history/fish_history

```diff
@@ -0,0 +1,11 @@
+[38;2;248;248;242m-[0m[38;2;248;248;242m [0m[38;2;249;38;114mcmd[0m[38;2;248;248;242m:[0m[38;2;248;248;242m [0m[38;2;230;219;116mecho "hello world"[0m
```

Comment:
Thank you for taking the time to open a PR!

I'm a bit confused, because this file looks very gray and not very YAML-y. Do you see what I see or am I doing something wrong?

<img width="488" alt="Screenshot 2022-08-12 at 15 30 11" src="https://user-images.githubusercontent.com/115040/184363973-3933510d-07a5-4108-a480-8df98048ab97.png">

Reply:
The file looks fine to me.

Reply:
```
curl -q https://raw.githubusercontent.com/sharkdp/bat/1e1fa7cde5c973ea42c97ea18479d1b0b1d3d4a0/tests/syntax-tests/highlighted/fish_history/fish_history
```
![image](https://user-images.githubusercontent.com/4209276/188332121-89bba75f-019f-48d0-84b3-787713a79a02.png)